### PR TITLE
Add ansible plabook

### PR DESCRIPTION
**Pull request by yishaiz**
install python by ansible playbook

### CODE REVIEW

File: provisioning/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  hosts: all
+  connection: local
+  user: root
+  become: yes
+
+
+  tasks:
+  - name:  Install Python3 package
+    package:  
+      name=python3
```
Comment (bregman-arie) [inline provisioning/playbook.yml]: It's better to use the convention:

name: 'python3'
state: 'latest'

File: provisioning/playbook.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+- name: Install the latest git package
+  # hosts: localhost
+  hosts: all
+  connection: local
+  user: root
```
Comment (bregman-arie) [inline provisioning/playbook.yml]: become: yes is by default root. No need to specify user: root

File: provisioning/playbook.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  become: yes
+
+
+  tasks:
+  - name:  Install Python3 package
```
Comment (bregman-arie) [inline provisioning/playbook.yml]: Missing indention

**Comment by noamgu** [comment]
@yishaiz let's expend this playbook to replace (at least) of on our bootstrap scripts.
bootstrap-infra.sh Should be preaty simple to do.

File: provisioning/playbook.yml (hunk 1 above)
Comment (yishaiz) [reply provisioning/playbook.yml]: Thanks a lot for feedback !
Anyway i rollback this PR because someone already did this task.

**Comment by yishaiz** [comment]
I close this becuase Yossi already has a progress with this task.

**Pull request closed without merging**